.ci/build.sh

```diff
@@ -23,10 +23,11 @@ if [[ "$GHC_HEAD" != "yes" ]]; then
 fi
 set -u
 
-cabal v2-build clash-prelude --write-ghc-environment-files=always
-# Undo mmap crash workaround for clash-cosim, see PR #2572.
-# It uses a custom setup, which can't be linked with -rtsopts
-GHCRTS="" cabal v2-build clash-cosim --write-ghc-environment-files=always
+# Any package with a custom setup can only be build with GHCRTS=""
+# see PR #2572.
+GHCRTS="" cabal v2-build ghc-paths
```

Comment:
This will also build `base` without `GHCRTS`, isn't it generally better to do
```sh
cabal v2-build --only-dependencies ghc-paths
GHCRTS="" cabal v2-build ghc-paths
```

Reply:
Ah, also _can only be build_ should of course be _built_ :-)

Reply:
But I've done the 2-step thing now, because I had to add more packages